client/components/PageLayout.js

```diff
@@ -0,0 +1,16 @@
+import { Card, Layout, Page } from "@shopify/polaris";
+import React from "react";
+
+function PageLayout({ children }) {
+  return (
+    <Page>
+      <Layout>
+        <Layout.Section>
+          <Card>{children}</Card>
+        </Layout.Section>
```

Comment:
ah hmm 🤔 but it looks bad otherwise 
![image](https://user-images.githubusercontent.com/30154753/139740466-0d633bfb-aeb5-4b00-9353-78cf3ad61ad0.png)